code/modules/reagents/holder.dm

```diff
@@ -493,8 +493,44 @@
 	chem_temp = round(chem_temp)
 	handle_reactions()
 
+//Returns the average specific heat for all reagents currently in this holder.
+/datum/reagents/proc/specific_heat()
+	/*
+	. = 0
+	var/cached_amount = total_volume		//cache amount
+	var/list/cached_reagents = reagent_list		//cache reagents
+	for(var/I in cached_reagents)
+		var/datum/reagent/R = I
+		. += R.specific_heat * (R.volume / cached_amount)
+	*/
```

Comment:
More commented out code.

Reply:
See the line below. It's there in case reagent specific heat ever gets ported.